ts/components/session/RegistrationTabs.tsx

```diff
@@ -713,13 +713,8 @@ export class RegistrationTabs extends React.Component<{}, State> {
   }
 
   private async resetRegistration() {
-    await window.Signal.Data.removeAllIdentityKeys();
-    await window.Signal.Data.removeAllPrivateConversations();
-    window.Whisper.Registration.remove();
-    // Do not remove all items since they are only set
-    // at startup.
-    window.textsecure.storage.remove('identityKey');
-    window.textsecure.storage.remove('secondaryDeviceStatus');
```

Comment:
a note about why these aren't/are removed would be good in the PR

Reply:
Removes our identity key and status. This was removed because they get stored as `items` in the db. `removeAll` removes all items anyway